src/commands/pascalify.ts

```diff
@@ -0,0 +1,11 @@
+import { Message } from 'discord.js';
+
+import { Command } from '../utils/commandHandler';
+
+import PascalCase from 'pascal-case';
+
+export const command = new Command({
+    description: 'Converts user message to PascalCase',
+    aliases: ['PascalCase', 'Pascalify', 'pascalify', 'Robinify'],
+    command: async (message: Message): Promise<Message> => message.channel.send(PascalCase(message.content)),
```

Comment:
this.. dosent return a message

Reply:
How? message.channel.send has Promise<Message> as return type?

Reply:
ahh nvm, i thought there were braces haha